app/assets/javascripts/main.js.erb

```diff
@@ -706,10 +764,11 @@ $(function() {
           type: 'GET',
           url: '/info_window',
           data: {
-            'thing_id': activeThingId,
+            'thing_id': activeThingId
+            /*
```

Comment:
Can you throw some `*`s on the commented lines just to make it stand out more that those lines are commented?